packages/RatPack.package/RPModel.class/class/getUpdatedObject.withJson..st

```diff
@@ -0,0 +1,10 @@
+object conversion
+getUpdatedObject: aRPModelClassObject withJson: aJsonString
+
+	| newObject aJsonObject|
+	
+	aJsonObject := self createUpdateObject: aJsonString. 
+	newObject := aRPModelClassObject copy.  
```

Comment:
Why do we need a copy here?

Reply:
I just disliked the idea of modifying objects / instances by class methods. So I wanted an encapsulation and not modifying the instance itself rather a copy.
But I accept that it is completely irrelevant, because it is ugly anyway and we should change this.